storage/postgres/util/data-mapping.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+/**
+ * Get Mapped data
+ *
+ * Converts data into a format to be stored in database
+ *
+ * @param {object} data - data to be converted
+ *
+ * @returns {object} - object in required db format
+ */
+const getMappedData = (data) => {
+  const mappedData = {
+    userEmail: data['user-email'],
+    jsonSavedData: data,
+  };
+
+  return mappedData;
+};
+
+module.exports = {
+  getData: getMappedData,
+};
```

Comment:
This file is going to be used convert our app data to a format for the database to injest @sulthan-ahmed